src/Update.hs

```diff
@@ -223,8 +227,14 @@ updatePackage log updateEnv mergeBaseOutpathsContext =
     Nix.build attrPath
     newSrcUrl <- Nix.getSrcUrl attrPath
     --
-    result <- Nix.resultLink
-    publishPackage log updateEnv oldSrcUrl newSrcUrl attrPath result opDiff
+    -- Either publish the result, or just print a diff and quit
+    if (dryRun o)
+      then do
+        gitDiff <- Git.diff
+        lift . log $ "Successfully finished processing, rewrote derivation with diff:\n" <> gitDiff
+      else do
+        result <- Nix.resultLink
+        publishPackage log updateEnv oldSrcUrl newSrcUrl attrPath result opDiff
```

Comment:
Result on running a dry-run looks like this in the logfile:
```
2020-02-20T04:48:51 New run of nixpkgs-update
2020-02-20T04:48:51 jruby 9.2.9.0 -> 9.2.10.0
2020-02-20T04:51:57 [version] started
2020-02-20T04:52:03 [version]: updated version and sha256
2020-02-20T04:52:03 [quotedUrls] started
2020-02-20T04:52:03 [quotedUrls]: added quotes to meta.homepage
2020-02-20T04:55:09 Successfully finished processing, rewrote derivation with diff:
diff --git a/pkgs/development/interpreters/jruby/default.nix b/pkgs/development/interpreters/jruby/default.nix
index 4e3f92d7cdc..e9eaaca4730 100644
--- a/pkgs/development/interpreters/jruby/default.nix
+++ b/pkgs/development/interpreters/jruby/default.nix
@@ -6,11 +6,11 @@ rubyVersion = callPackage ../ruby/ruby-version.nix {} "2" "3" "3" "";
 jruby = stdenv.mkDerivation rec {
   pname = "jruby";

-  version = "9.2.9.0";
+  version = "9.2.10.0";

   src = fetchurl {
     url = "https://s3.amazonaws.com/jruby.org/downloads/${version}/jruby-bin-${version}.tar.gz";
-    sha256 = "04grdf57c1dgragm17yyjk69ak8mwiwfc1vjzskzcaag3fwgplyf";
+    sha256 = "0lwmqyg06rv3iyhmfnqcg4xzaxcfrgjivjrc4ljwb0y629vp16ci";
   };

   buildInputs = [ makeWrapper ];
@@ -48,7 +48,7 @@ jruby = stdenv.mkDerivation rec {

   meta = {
     description = "Ruby interpreter written in Java";
-    homepage = http://jruby.org/;
+    homepage = "http://jruby.org/";
     license = with stdenv.lib.licenses; [ cpl10 gpl2 lgpl21 ];
     platforms = stdenv.lib.platforms.unix;
   };

2020-02-20T04:55:09 SUCCESS

```

Reply:
You can see this did the same as https://github.com/NixOS/nixpkgs/pull/80600, except it also quoted the homepage :)